src/qnetvo/network_ansatz.py

```diff
@@ -256,7 +256,7 @@ def qnode_settings(self, scenario_settings, prep_inputs, meas_inputs):
 
         prep_settings = self.layer_settings(scenario_settings[0], prep_inputs, self.prepare_nodes)
         meas_settings = self.layer_settings(scenario_settings[1], meas_inputs, self.measure_nodes)
-        return math.concatenate([prep_settings, meas_settings])
+        return np.concatenate([prep_settings, meas_settings])
```

Comment:
I was expecting for this to have some implications for differentiability :thinking:

Reply:
me too, I thought the tensorflow tests were gonna fail. But maybe this was never a problem